src/App.jsx

```diff
@@ -14,70 +15,58 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const defaultSelect = 'No goods selected';
+  const firstSelected = 'Jam';
+  const [value, setValue] = useState(firstSelected);
```

Comment:
there should be stored selected good or `null`
do not save here `'No goods selected'` message